spec/components/validators/post_validator_spec.rb

```diff
@@ -57,6 +57,18 @@
       validator.stripped_length(post)
       expect(post.errors.count).to eq(0)
     end
+
+    it "ignores an html comment" do
+      post.raw = "<!-- an html comment -->abc"
+      validator.stripped_length(post)
+      expect(post.errors.count).to eq(1)
+    end
+
+    it "ignores multiple html comments" do
```

Comment:
What if html comments are nested?

Reply:
You can't really nest them, the first `-->` will end the comment. I added some tests to be sure.

Reply:
Sounds good